utils.js

```diff
@@ -1,20 +1,32 @@
+const alfy = require('alfy');
 const moment = require('moment');
 
-const fetchOptions = (token, query = '') => ({
-    headers: {
-        Authorization: `Bearer ${token}`
-    },
-    method: 'GET',
-    query: {
+const fetchOptions = ({ token, query, maxAge = 0, sort = '-updated_on', fields }) => {
+    const maxAgeMillis = maxAge * 60 * 1000;
+
+    const queryOptions = {
         q: (query) ? `name~"${query}"` : '',
-        sort: '-updated_on',
+        sort,
         role: 'member'
+    };
+
+    if (fields) {
+        queryOptions.fields = fields;
```

Comment:
You could add this directly to the `queryOptions` object using or operator like 

```
const queryOptions = {
  //...
  fields: fields || undefined,
}
```

If I remember well, undefined values should be removed from the object when calling.